backend/coins/eth/account.go

```diff
@@ -738,18 +749,19 @@ func (account *Account) TxProposal(
 
 // GetUnusedReceiveAddresses implements accounts.Interface.
 func (account *Account) GetUnusedReceiveAddresses() []accounts.AddressList {
+	if !account.isInitialized() {
+		return nil
+	}
 	return []accounts.AddressList{{
 		Addresses: []accounts.Address{account.address},
 	}}
 }
 
 // VerifyAddress implements accounts.Interface.
 func (account *Account) VerifyAddress(addressID string) (bool, error) {
-	if account.signingConfiguration == nil {
+	if !account.isInitialized() {
 		return false, errp.New("account must be initialized")
 	}
-	account.Synchronizer.WaitSynchronized()
```

Comment:
Not sure why this is not needed anymore. Could be also removed from `backend/coins/btc/account.go` in https://github.com/digitalbitbox/bitbox-wallet-app/pull/1876/ ?

Reply:
It is not needed because a few lines above we check if the account is initialized (now with `if !account.isInitialized()`, before with `if account.signingConfiguration == nil`. Waiting for transactions/balance etc to sync is not needed to be able to verify the address.

Need to take a closer look in the btc account, as there, the address list can change during synchronization. The goal is definitely to remove all blocking `WaitSynchronized()` and replace them with error responses however, so it will go away one way or another :)